integration-libs/digital-payments/src/digital-payments.module.ts

```diff
@@ -1,6 +1,7 @@
-import { DpCheckoutModule } from './checkout/dp-checkout.module';
 import { NgModule } from '@angular/core';
+import { CheckoutModule } from '@spartacus/checkout';
+import { DpCheckoutModule } from './checkout/dp-checkout.module';
 @NgModule({
-  imports: [DpCheckoutModule],
+  imports: [CheckoutModule, DpCheckoutModule],
```

Comment:
Just a question:  Is this an example of what would be a "built-in" wrapper module?

I thought we had to create an extra module to be the wrapper.

Reply:
This is can be a wrapper module, because `DigitalPaymentsModule` itself is a wrapper module wrapping `DpCheckoutModule`.  If we create another wrapper, then there will be a wrapper inside another wrapper.